File: JSBASIC/111_factorial.js
// 111	111_factorial.js	Find the factorial of a given number using a loop.

let num = 6;
let factorial = 1;
for (let i = 1; i <= num; i++) {
    factorial *= i;
}
console.log("Number:", num);
console.log("Factorial:", factorial);

File: JavaBasic/111_factorial.java
// 111	111_factorial.java	Find the factorial of a given number using a loop.
public class _111_factorial {
    public static void main(String[] args) {
        int num = 6;
        long factorial = 1;
        for (int i = 1; i <= num; i++) factorial *= i;
        System.out.println(num + "! = " + factorial);
    }
}
